src/Components/ApiCalls.js

```diff
@@ -1,6 +1,20 @@
+// export function getMovies() {
+//   return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+//   .then(res => res.json())
+//   // .then(data => data)
+// }
+
 export function getMovies() {
   return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-  .then(res => res.json())
-  // .then(data => data)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.status}`);
+      }
+      return response.json();
+    })
+    .catch(error => {
+      console.error("There was a problem with the fetch operation:", error);
+    });
```

Comment:
Good error handling! We could possibly move this into the useEffect hook, but for now this is great!